diff --git a/node_modules/@crxjs/vite-plugin/dist/index.mjs b/node_modules/@crxjs/vite-plugin/dist/index.mjs
index 604cfbc..7286c88 100644
--- a/node_modules/@crxjs/vite-plugin/dist/index.mjs
+++ b/node_modules/@crxjs/vite-plugin/dist/index.mjs
@@ -794,7 +794,7 @@ const pluginDeclaredContentScripts = () => {
   return [];
 };
 
-const _dynamicScriptRegEx = /\b(import.meta).CRX_DYNAMIC_SCRIPT_(.+?)[,;]/gm;
+const _dynamicScriptRegEx = /\b(import.meta).CRX_DYNAMIC_SCRIPT_([a-zA-Z0-9_$]+)/gm;
 const dynamicScriptRegEx = () => {
   _dynamicScriptRegEx.lastIndex = 0;
   return _dynamicScriptRegEx;
@@ -889,6 +889,7 @@ const pluginDynamicContentScripts = () => {
                 matches: []
               });
               contentScripts.set(script.id, script);
+              if (script.refId) contentScripts.set(script.refId, script);
             }
             return resolvedId;
           } else if (url.searchParams.has("scriptId")) {
@@ -1478,7 +1479,7 @@ var loadingPageHtml = "<!DOCTYPE html>\n<html lang=\"en\">\n  <head>\n    <title
 const { readFile } = promises;
 const pluginManifest = () => {
   let manifest;
-  let plugins;
+  let plugins = [];
   let refId;
   let config;
   return [
